test/navigator/get.spec.js

```diff
@@ -123,4 +123,40 @@ describe('Navigator', () => {
       'Mary'
     ])
   })
+
+  it('should be able to add header options for navigation', async () => {
+    api.onDiscover(baseUrl, {
+      users: { href: '/users{?admin}', templated: true }
+    })
+
+    const headers = {
+      authorization: 'some-token'
+    }
+
+    api.onGet(baseUrl, '/users',
+      new Resource()
+        .addResource('users', [
+          createUser({ id: 'fred', name: 'Fred' }),
+          createUser({ id: 'sue', name: 'Sue' }),
+          createUser({ id: 'mary', name: 'Mary' })
+        ]), headers)
+
+    const discoveryResult = await Navigator.discover(baseUrl)
+    const result = await discoveryResult.get('users', {}, {headers})
```

Comment:
Was hoping the linter would catch this, but it didn't.

Can you put spacing around the braces? e.g. `{headers}` -> `{ headers }`